packages/core/src/browser/about-dialog.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2018 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+import { Message } from '@phosphor/messaging';
+import { AbstractDialog } from './dialogs';
+import { ExtensionInfo, ApplicationInfo } from '../common/application-protocol';
+
+export const ABOUT_CONTENT_CLASS = 'theia-aboutDialog';
+export const ABOUT_EXTENSIONS_CLASS = 'theia-aboutExtensions';
+
+export class AboutDialog extends AbstractDialog<void> {
+
+    protected readonly okButton: HTMLButtonElement;
+
+    constructor(extensionsInfos: ExtensionInfo[], applicationInfo?: ApplicationInfo) {
```

Comment:
I feel like I should pass the application-server directly instead of the infos here...

Reply:
Yes or inject it.

Reply:
If I inject it I have to make the whole about-dialog injectable. Then I need to bind some DialogProps and whatnot no?

Reply:
I've pushed a new patch with the server proxy injected, let me know what you think.